jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpException.java

```diff
@@ -0,0 +1,123 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.http;
+
+import org.eclipse.jetty.io.QuietException;
+import org.eclipse.jetty.util.ExceptionUtil;
+
+/**
+ * <p>A tagging interface for Exceptions that carry a HTTP response code and reason.</p>
+ * <p>Exception sub-classes that implement this interface will be caught by the container
+ * and the {@link #getCode()} used to send a response.</p>
+ */
+@SuppressWarnings("serial")
```

Comment:
What's with this suppressions? Just remove them, these exceptions are not serializable anyway and we don't want them to be.

Reply:
No idea. They were in the code to start with.